src/Symfony/Component/Mailer/Bridge/Postmark/Transport/PostmarkApiTransport.php

```diff
@@ -51,13 +54,26 @@ public function __toString(): string
 
     protected function doSendApi(SentMessage $sentMessage, Email $email, Envelope $envelope): ResponseInterface
     {
-        $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/email', [
-            'headers' => [
-                'Accept' => 'application/json',
-                'X-Postmark-Server-Token' => $this->key,
-            ],
-            'json' => $this->getPayload($email, $envelope),
-        ]);
+        try {
+            $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/email', [
+                'headers' => [
+                    'Accept' => 'application/json',
+                    'X-Postmark-Server-Token' => $this->key,
+                ],
+                'json' => $this->getPayload($email, $envelope),
+            ]);
+        } catch (HttpTransportException $e) {
+            if (
+                str_contains($e->getMessage(), 'Unable to send an email: You tried to send to recipient(s) that have been marked as inactive.')
```

Comment:
this should be detected based on the error code provided by the postmark response instead of detecting that based on the exact wording of the human-readable error message. See https://postmarkapp.com/developer/api/overview#error-codes